build/container/oracle.Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Oracle
+# See https://hub.docker.com/r/gvenzl/oracle-free/tags
+FROM docker.io/gvenzl/oracle-free:23.3-slim-faststart
```

Comment:
Note that I've switched to `free` from `xe` as I've noticed that Quarkus and Reactive already switched 😃